src/main/java/com/softserveinc/ita/pageobjects/ProductPage.java

```diff
@@ -1,12 +1,15 @@
 package com.softserveinc.ita.pageobjects;
 
+import com.softserveinc.ita.models.Product;
+import com.codeborne.selenide.SelenideElement;
+import com.softserveinc.ita.pageobjects.modals.BasketModal;
 import io.qameta.allure.Step;
 
 import static com.codeborne.selenide.Selenide.$x;
 import static com.codeborne.selenide.Condition.visible;
 import static com.codeborne.selenide.Selenide.$;
 
-import static com.softserveinc.ita.pageobjects.ProductModel.*;
+import static com.softserveinc.ita.utils.NumberUtil.parseIntPrice;
 import static java.lang.String.format;
 import static java.time.Duration.ofSeconds;
 
```

Comment:
take a look onto getCurrentPriceFromFirstItem if it should be in this page object

Reply:
Actually - yes, because we are taking price from first item on products page

Reply:
nope, for my opinion it's search results page
![image](https://user-images.githubusercontent.com/21026576/146793227-d6951d2e-0a28-4532-a182-18e68a8c4bab.png)